webpack/assets/javascripts/react_app/components/UserJwt/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { post } from '../../redux/API';
+import {
+  selectAPIByKey,
+  selectAPIResponse,
+} from '../../redux/API/APISelectors';
+import { foremanUrl } from '../../../foreman_tools';
+
+import { USER_JWT } from './UserJwtConstants';
+import UserJwtForm from './components/UserJwtForm';
+import UserJwtField from './components/UserJwtField';
+
+const UserJwt = () => {
+  const dispatch = useDispatch();
+  const apiStatus = useSelector(
+    state => selectAPIByKey(state, USER_JWT).status || null
+  );
+  const token = useSelector(
+    state => selectAPIResponse(state, USER_JWT).jwt || null
+  );
+
+  const handleSubmit = (expirationUnit, expirationValue) => {
+    const params = {
+      expiration_unit: expirationUnit,
+      expiration_value: expirationValue,
+    };
+
+    dispatch(
+      post({
+        key: USER_JWT,
+        url: foremanUrl('/users/jwt'),
+        params,
+      })
+    );
+  };
+
+  return (
+    <div>
+      <UserJwtForm handleSubmit={handleSubmit} />
+      <br />
```

Comment:
based on https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br
it became a bad practice to use `<br />` instead of css margin.